store/filestore.go

```diff
@@ -1,15 +1,16 @@
 package store
 
 import (
-	"github.com/djherbis/atime"
-	"github.com/kailt/imageresizer/collections"
 	"io/ioutil"
 	"math/rand"
 	"os"
 	"path/filepath"
 	"strings"
 	"sync/atomic"
 	"time"
+
```

Comment:
Cosmetic change unrelated to the fsnotify feature. Please revert. This file was already gofmt'ed.